CONTRIBUTING.md

```diff
@@ -90,6 +90,14 @@ If you are working in VSCode, we recommend you install the [rust-analyzer](https
 ```json
 "editor.formatOnSave": true,
 "rust-analyzer.rustfmt.extraArgs": ["+nightly"],
+"rust-analyzer.checkOnSave.overrideCommand": [
+  "cargo",
+  "+nightly",
+  "clippy",
+  "--all",
+  "--all-features",
+  "--message-format=json"
+],
```

Comment:
```suggestion
"rust-analyzer.checkOnSave.extraArgs": ["+nightly"],
```

Reply:
Hmmm I have a similar error:
```
[ERROR flycheck] Flycheck failed to run the following command: "cargo" "clippy" "--workspace" "--message-format=json" "--manifest-path" "/workspaces/crypto/learn/ethereum/reth/reth/Cargo.toml" "--all-targets" "--all-features" "+nightly"
[ERROR rust_analyzer::lsp_utils] cargo check failed

Cargo watcher failed, the command produced no valid metadata (exit code: ExitStatus(unix_wait_status(256))):
error: Found argument '+nightly' which wasn't expected, or isn't valid in this context
```

Reply:
Alright, strange. This is fine then, thank you for being patient!